policyengine_us_data/datasets/cps/cps.py

```diff
@@ -340,10 +341,13 @@ def add_personal_income_variables(
         year (int): The CPS year
     """
     # Get income imputation parameters.
-    yamlfilename = os.path.join(
-        os.path.abspath(os.path.dirname(__file__)),
-        "imputation_parameters.yaml",
+    yamlfilename = (
+        files("policyengine_us_data")
```

Comment:
This can be run interactively, which is good, but it still has the problem that it will be different if you are not in the repo developing. So you will get your enivornment location or your repo location, depending on where you are.

Not entirely sure what to do. That might just be a feature of development.

Reply:
Should at least document it.